### DISCUSSION TITLE
Show the transitionable states of the current state

### DISCUSSION THREAD

**Discussion by risky2k1** [spatie/laravel-model-states#239] [Q&A]
Im currently using :
       ` $states = collect(Order::getStates()->first())->toArray();`
to get all the state of the Order model
when using @foreach in view i can display each state($state).
With the $state, how can i get the array of the transitionable states of that $state?

**Comment by jeffchown** [comment]
If you still need this info, you can find it here: https://spatie.be/docs/laravel-model-states/v2/working-with-transitions/04-retrieving-transitionable-states
